Dependencies fail with python 3.11.4

on aiohttp==3.8.1

```
gcc -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/home/lyma/.pyenv/versions/3.11.4/envs/g2s/include -I/home/lyma/.pyenv/versions/3.11.4/include/python3.11 -c aiohttp/_websocket.c -o build/temp.linux-x86_64-cpython-311/aiohttp/_websocket.o
      aiohttp/_websocket.c:198:12: fatal error: longintrepr.h: No such file or directory
        198 |   #include "longintrepr.h"
            |            ^~~~~~~~~~~~~~~
      compilation terminated.
      error: command '/usr/bin/gcc' failed with exit code 1
      [end of output]

``

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,4 +1,5 @@
-aiohttp==3.8.1
+Unidecode==0.4.21
+aiohttp==3.8.6
 awesome-slugify==1.6.5
 certifi==2020.6.20
 chardet==3.0.4
@@ -8,8 +9,7 @@ python-gitlab==2.5.0
 pytz==2022.1
 pyyaml==6.0.0
 regex==2020.10.28
-requests==2.24.0
 requests-toolbelt==0.9.1
+requests==2.24.0
 sentry-sdk==0.19.5
-Unidecode==0.4.21
 urllib3==1.25.11

